aux/custom-first-boot.service

```diff
@@ -0,0 +1,21 @@
+[Unit]
+Description=Custom first boot script
+ConditionFileIsExecutable=/usr/local/sbin/custom-first-boot
+ConditionFirstBoot=yes
+
+Wants=first-boot-complete.target
+Wants=network-online.target
+Wants=osbuild-first-boot.target
+Before=first-boot-complete.target
+After=network-online.target
+After=osbuild-first-boot.service
+
+[Service]
+Type=oneshot
+ExecStart=/usr/local/sbin/custom-first-boot
+RemainAfterExit=yes
+
+[Install]
+WantedBy=basic.target
+WantedBy=multi-user.target
+WantedBy=graphical.target
```

Comment:
We have a report that when target is changed to `grahical.target` firstboot is disabled. This ensures it will always run.

Reply:
Good find! :orange_heart: